benchmarks/README.md

```diff
@@ -0,0 +1,23 @@
+# Torchdynamo Benchmarks
+
+## What We Benchmark
+TorchDynamo provides a benchmark harness that takes care of uniformly benchmarking different models.  It interleaves runs of eager and dynamo to avoid machine noise/variability issues, and reports results based on medians along with P-values. 
+
+The runner integrates with models from TorchBenchmark, HuggingFace and TIMM suites and covers both training and inference.
+
+Training benchmarks approximate training by running the model forward, using a .sum().backward() call in place of the native loss function and skips the optimizer entirely.
+
+Inference benchmarks and Training benchmarks measure correctness by comparing dynamo and eager model outputs given fixed inputs and seeds.
+
+## Runbook
+There are a lot of flags in the benchmark runner, and it can be confusing to know which settings to use or what machine to run it on.  In order to support apples-to-apples comparison, we have provided the following 'standard' settings.
+
+We run benchmarks on AWS machines using 8xNVidia A100 64GB cards and (TODO) CPU.  (TODO instance type and OS/cuda versions) 
+
+### Training
```

Comment:
@anijain2305 @jansel can you paste some commands in a comment and i'll add them here?

Reply:
(from offline chat - any objections to these?)
For nvfuser
`--training  --accuracy-aot-ts-mincut --inductor-settings --float16 --isolate --nvfuser`
(won't use cuda graphs though, don't know how to make it)

For inductor
`--training --inductor --float16 --isolate`
whether cudagraphs are used or not is controlled in torchinductor/config.py files, by default it's on.

Reply:
Sure, we can list flags specifically instead of using `--inductor-setting`.

Reply:
Hi Will, I am working on a PR that generates this commands automatically. I plan to use this to run commands, and then parse the results in some pretty form. The result of the command geeration is this

~~~
# Commands for torchbench for device=cuda, dtype=float32 for training
python benchmarks/torchbench.py --float32 -dcuda --output=ts_nvfuser_torchbench_float32_training_cuda.csv --training --nvfuser --speedup-ts --use-eval-mode --isolate
python benchmarks/torchbench.py --float32 -dcuda --output=aot_nvfuser_torchbench_float32_training_cuda.csv --training --nvfuser --accuracy-aot-ts-mincut --use-eval-mode --isolate

# Commands for huggingface for device=cuda, dtype=float32 for training
python benchmarks/huggingface.py --float32 -dcuda --output=ts_nvfuser_huggingface_float32_training_cuda.csv --training --nvfuser --speedup-ts --use-eval-mode --isolate
python benchmarks/huggingface.py --float32 -dcuda --output=aot_nvfuser_huggingface_float32_training_cuda.csv --training --nvfuser --accuracy-aot-ts-mincut --use-eval-mode --isolate

~~~

Reply:
For inductor:
```
--training --inductor --float16 -dcuda
```

Don't forget `-dcuda`.   Perhaps we should make `-dcuda` the default.